src/wovn_interceptor.php

```diff
@@ -22,8 +22,11 @@
 $_ENV['WOVN_TARGET_LANG'] = $headers->lang();
 $headers->requestOut();
 
-if (!Utils::isFilePathURI($headers->getDocumentURI(), $store) &&
-    !Utils::isIgnoredPath($headers->getDocumentURI(), $store)) {
+$uri = $headers->getDocumentURI();
+if (!Utils::isFilePathURI($uri, $store) &&
```

Comment:
I would be OK with moving `shouldIgnoreBySitePrefixPath` inside `isIgnoredPath`. A path is ignored if it does not match the prefix path, after all.

Reply:
Done